apps/assets/tasks/ping.py

```diff
@@ -48,3 +53,38 @@ def test_node_assets_connectivity_manual(node):
     asset_ids = [str(i) for i in asset_ids]
     org_id = str(current_org.id)
     return test_assets_connectivity_task.delay(asset_ids, org_id, task_name)
+
+
+@shared_task(
+    verbose_name=_('Test assets port connectivity'),
+    activity_callback=lambda self, asset_ids, port, *args, **kwargs: (asset_ids,),
+    description=_(
+        "When clicking 'Test Asset Port Connectivity' in 'Asset List' this task will be executed"
+    )
+)
+def test_assets_port_connectivity(asset_ids, port):
+    log_path = get_celery_task_log_path(test_assets_port_connectivity.request.id)
+    printer = ColoredFilePrinter(open(log_path, 'a'), flush_now=True)
+    assets = Asset.objects.filter(id__in=asset_ids).values('name', 'address')
+    msg = f"{_('Asset')}: %s {_('Test port')} {port}: %s"
+    success, failed = [], []
+    for asset in assets:
+        asset_display = f'{asset["name"]}({asset["address"]})'
+        try:
+            socket.create_connection(
+                (asset["address"], port), timeout=10, all_errors=True
+            )
+            printer.green(msg % (asset_display, _('Success')))
+            success.append(asset)
+        except ExceptionGroup:
+            printer.red(msg % (asset_display, _('Failed')))
+            failed.append(asset)
+    printer.line()
+    printer.info(
+        _('Total {}, success {}, failure {}').format(len(assets), len(success), len(failed))
+    )
+
+
+def test_assets_port_connectivity_manual(assets, port):
+    asset_ids = [str(i.id) for i in assets]
+    return test_assets_port_connectivity.delay(asset_ids, port)
```

Comment:
It seems like the code you provided is missing some details but it's mostly clear. The main differences seem to be related to:

- A typo on `get_logger`:
    - In line 3, `"get_logger"` should be replaced with `"get_logger()"`.
- Different function names (e.g., tests for connecting nodes instead of assets): 
    - These have been removed since they refer to functionality not covered by your example.
  
If there are further differences to address that need explanation or verification I'd recommend checking the full context in both places mentioned.

For consistency throughout the project, ensure each variable name, logging setup, and any imports match what has been used before or if adjustments are necessary based on new data or features added.

Lastly,
- It'd be a bit more useful if these were part of an existing codebase where other team members might find value in understanding them.
- Also, adding comments/indents around lines could be helpful. This would make the code easier to follow even when reviewing long chunks of code at once.

The overall quality appears good, as many of the changes focus on clarifications/revisions rather than major structural modifications which can often introduce complexity or break backward compatibility.

Keep in mind that without additional context about specifics like environment settings, testing strategies etc... certain aspects could only really be addressed through deeper analysis.